### ISSUE TITLE
Differences in onehostname article vs. code

### ISSUE THREAD

**Issue by OKNoah** [superfly/onehostname#7]
I read **[this article](https://fly.io/articles/one-hostname-to-rule-them-all/)**, early on and it helped introduce me to fly.io, now one of my favourite startups (as you should know!)

I also looked at the [source code](https://github.com/superfly/onehostname) (I reported that it didn't work because it's in typescript and the instructions didn't account for that, ages ago, believe it's fixed).

But I'm back on my bullshit and want to say that **the article doesn't match up with the source-code** when it comes to endpoint names. Yeah, I skimmed it the second or third time I read it, while volunteering some time to make fly.io examples for fly and it's users.

Authors can be at liberty to give example endpoints, but I confused myself when writing examples because I looked at the article.

But maybe the article could include the same endpoints as the sourcecode? Are we open to a PR for that?

Also just noticed from the top of the [fly.io/article](https://fly.io/articles/one-hostname-to-rule-them-all/) link near the bottom there is [this link](https://fly.io/mix/edge-applications) which doesn't currently work.

**Comment by mrkurt** [comment]
Yeah we're open to PRs. I think the README for this repo could be a new version of that article. It's different right now because that article is old and for our "sites", which are going away.

**Comment by OKNoah** [comment]
@mrkurt you can assign this to me, i could get around to it
